dev/com.ibm.ws.runtime.update/resources/com/ibm/ws/runtime/update/internal/resources/Messages.nlsprops

```diff
@@ -61,4 +61,12 @@ quiesce.listeners.not.complete.useraction=No action is required.
 
 quiesce.waiting.on.threads=CWWKE1107W: {0} threads did not complete during the quiesce period. 
 quiesce.waiting.on.threads.explanation=One or more threads that were active before shutting down the server were still active at the end of the quiesce period.
-quiesce.waiting.on.threads.useraction=No action is required. 
\ No newline at end of file
+quiesce.waiting.on.threads.useraction=No action is required.
+
+quiesce.timeout.not.available=CWWKE1108W: The quiesce timeout value could not be obtained from the ''ApplicationManager''.  The default quiesce timeout value of ''{0}'' will be used instead. 
+quiesce.timeout.not.available.explanation=The 'RuntimeUpdateMangager' was not able to retrieve the 'quiesceTimeout' value from the 'ApplicationManager'.    
+quiesce.timeout.not.available.useraction=Possibly the 'ApplicationManager' did not start because of a confuration problem. Check the configuration of the 'applicationManager' element in the server.xml.  Ensure all of the attributes are configured correctly.
```

Comment:
```suggestion
quiesce.timeout.not.available.useraction=Possibly, the 'ApplicationManager' did not start because of a configuration problem. Check the configuration of the 'applicationManager' element in the 'server.xml' file.  Ensure all of the attributes are configured correctly.
```